ui-components/src/locales/ar.json

```diff
@@ -571,6 +571,13 @@
         "viewApplication": "مشاهدة التطبيق",
         "yourLotteryNumber": "رقم التأكيد الخاص بك هو"
     },
+    "users": {
```

Comment:
It looks like this change is based on top of your change to update all the strings? Please rebase once that's in so that this only has the new strings.

Reply:
Done.